WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md

```diff
@@ -0,0 +1,401 @@
+ms.author: roharwoo
+ms.date: 01/18/2023
+---
+
+# Configure trusted roots and disallowed certificates
+
+Applies To: Windows Server (All supported versions), Windows clients, Azure Stack HCI.
+
+This article contains guidance intended to help administrators install automatic daily update functionality of Certificate Trust Lists (CTLs). Servers and clients access the Windows Update site to update CTLs using the automatic daily Root Certificate Updates mechanism discussed in this article.
+
+See [Certificates and trust](certificates-and-trust.md) for more detailed knowledge on how the Microsoft Root Certificate Program works to distribute trusted root certificates automatically across Windows operating systems.
+
+## Prerequisites
+
+- Client machine with internet access to run certutil
+- File server or web server for file storage
+- AD Group policy or MDM solution to deploy configuration settings to your client
+
+## Software update description
+
+The following automatic update mechanisms are available in Windows Server 2012 R2 and Windows 8.1 and newer Windows versions or when the appropriate software update is installed (for older operating systems discussed in this article):
+
+- **Registry settings for storing CTLs** New settings enable changing the location for uploading trusted or untrusted CTLs from the Windows Update site to a shared location in an organization. For more information, see the [Registry settings modified]() section.
+
+- **Synchronization options** If the URL for the Windows Update site is moved to a local shared folder, the local shared folder must be synchronized with the Windows Update folder. This software update adds a set of options in the Certutil tool that administrators can use to enable synchronization. For more information, see the [New Certutil Options]() section.
+
+- **Tool to select trusted root certificates** This software update introduces a tool for administrators who manage the set of trusted root certificates in their enterprise environment. Administrators can view and select the set of trusted root certificates, export them to a serialized certificate store, and distribute them by using Group Policy. For more information, see the [New Certutil Options]() section in this document.
+
+- **Independent configurability** The automatic update mechanism for trusted and untrusted certificates are independently configurable. Administrators can use the automatic update mechanism to download only the untrusted CTLs and manage their own list of trusted CTLs. For more information, see the [Registry settings modified]() section in this document.
+
+Automatic update functionality also can be disabled if necessary, although this isn't recommended. See [Event ID 8 — Automatic Root Certificates Update Configuration](https://learn.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc734054(v=ws.10)?redirectedfrom=MSDN#turn-off-automatic-root-certificates-update) for more information.
```

Comment:
although **this** isn't recommended. - This isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?